### ISSUE TITLE
Console error

### ISSUE THREAD

**Issue by chineyk** [bitwarden/browser#2365]
### Steps To Reproduce

Currently tested on Brave browser. Open any site when logged in to the extension and then check the console. I get multiple of these errors `DevTools failed to load source map: Could not load content for chrome-extension://nngceckbapebfimnlniiiahkandclblb/content/shortcuts.js.map: HTTP error: status code 404, net::ERR_UNKNOWN_URL_SCHEME`

### Expected Result

N/A

### Actual Result

N/A

### Operating System

Windows

### Web Browser

Brave

### Browser Version

Version 1.35.101 

### Build Version

Version: 1.56.1

**Comment by Brendito** [comment]
<img width="777" alt="image" src="https://user-images.githubusercontent.com/47006907/153683308-c3274056-e444-46f4-8f80-517b775f8e86.png">

Seeing this issue as well, here is my console.

v1.56.1

**Comment by iamandrewluca** [comment]
<img width="1440" alt="image" src="https://user-images.githubusercontent.com/1881266/153683619-b6278d81-0b3d-4b52-bb32-2bffa733a828.png">

**Comment by Hinton** [comment]
Hi, thanks for reporting this.

I've managed to reproduce it in the production released extension, and while the warnings can be safely ignored they do clutter up the console of the devtools a fair bit.

We will be looking into getting this resolved beginning next week.

**Comment by nicolaerario** [comment]
I can confirm that bug is present on chrome 1.56.3 also

**Comment by martindines** [comment]
And 1.56.4

```
DevTools failed to load source map: Could not load content for chrome-extension://nngceckbapebfimnlniiiahkandclblb/content/autofill.js.map: HTTP error: status code 404, net::ERR_UNKNOWN_URL_SCHEME
DevTools failed to load source map: Could not load content for chrome-extension://nngceckbapebfimnlniiiahkandclblb/content/autofiller.js.map: HTTP error: status code 404, net::ERR_UNKNOWN_URL_SCHEME
DevTools failed to load source map: Could not load content for chrome-extension://nngceckbapebfimnlniiiahkandclblb/content/notificationBar.js.map: HTTP error: status code 404, net::ERR_UNKNOWN_URL_SCHEME
DevTools failed to load source map: Could not load content for chrome-extension://nngceckbapebfimnlniiiahkandclblb/content/contextMenuHandler.js.map: HTTP error: status code 404, net::ERR_UNKNOWN_URL_SCHEME
DevTools failed to load source map: Could not load content for chrome-extension://nngceckbapebfimnlniiiahkandclblb/content/shortcuts.js.map: HTTP error: status code 404, net::ERR_UNKNOWN_URL_SCHEME
DevTools failed to load source map: Could not load content for chrome-extension://nngceckbapebfimnlniiiahkandclblb/content/message_handler.js.map: HTTP error: status code 404, net::ERR_UNKNOWN_URL_SCHEME
```

**Comment by Hinton** [comment]
I've opened a PR which should resolve these warnings.

Just a note for future references. the current stable chrome cannot load source maps within popups and background pages. This will produce a warning when using the devtools of the production build in either. However it's working in the current chrome canary which means it should land in stable after a while.

**Comment by andreaslarssen** [comment]
Great. This is really annoying

**Comment by alcohol** [comment]
I believe the fix from @Hinton does not address the following since they're not from "content" scripts (but rather from the bar that shows after a new/changed password form submission):

```
DevTools failed to load source map: Could not load content for chrome-extension://nngceckbapebfimnlniiiahkandclblb/notification/bar.js.map: HTTP error: status code 404, net::ERR_UNKNOWN_URL_SCHEME
13:34:53.448 DevTools failed to load source map: Could not load content for chrome-extension://nngceckbapebfimnlniiiahkandclblb/notification/bar.css.map: HTTP error: status code 404, net::ERR_UNKNOWN_URL_SCHEME
```

**Comment by Hinton** [comment]
@alcohol Correct. Seems we forgot to exclude the notification scripts. It should be addressed in #2388, and will be submitted to the Chrome store shortly for review.
